benchmark/opperf/nd_operations/nn_activation_operators.py

```diff
@@ -55,55 +55,106 @@ def run_activation_operators_benchmarks(ctx=mx.cpu(), dtype='float32', profiler=
+                                                dtype=dtype,
+                                                ctx=ctx,
+                                                profiler=profiler,
+                                                inputs=[{"data": (2**16, 2**16), "act_type": "leaky", "slope": 0.1},
+                                                        {"data": (2**4, 2**28), "act_type": "leaky", "slope": 0.1},
+                                                        {"data": (4, 2**30), "act_type": "leaky", "slope": 0.1},
+                                                        {"data": (2**16, 2**16), "act_type": "elu", "slope": 0.1},
+                                                        {"data": (2**4, 2**28), "act_type": "elu", "slope": 0.1},
+                                                        {"data": (4, 2**30), "act_type": "elu", "slope": 0.1},
+                                                        {"data": (2**16, 2**16), "act_type": "selu"},
+                                                        {"data": (2**4, 2**28), "act_type": "selu"},
+                                                        {"data": (4, 2**30), "act_type": "selu"},
+                                                        {"data": (2**16, 2**16), "act_type": "prelu", "gamma": (1, 2**16)},
+                                                        {"data": (2**4, 2**28), "act_type": "prelu", "gamma": (1, 2**28)},
+                                                        {"data": (4, 2**30), "act_type": "prelu", "gamma": (1, 2**30)}
+                                                        ],
+                                                warmup=warmup,
+                                                runs=runs)
 
-    # Sigmoid => Covered as part of Unary ops
-    # Hard_Sigmoid
-    hard_sigmoid_benchmark_res = run_performance_test([getattr(MX_OP_MODULE, "hard_sigmoid")],
-                                                      run_backward=True,
-                                                      dtype=dtype,
-                                                      ctx=ctx,
-                                                      profiler=profiler,
-                                                      inputs=[{"data": (1024, 1024), "alpha": 0.25, "beta": 0.5},
-                                                              {"data": (10000, 1), "alpha": 0.25, "beta": 0.5},
-                                                              {"data": (10000, 100), "alpha": 0.25, "beta": 0.5}
-                                                              ],
-                                                      warmup=warmup,
-                                                      runs=runs)
+        # Sigmoid => Covered as part of Unary ops
+        # Hard_Sigmoid
+        hard_sigmoid_benchmark_res = run_performance_test([getattr(MX_OP_MODULE, "hard_sigmoid")],
+                                                        run_backward=True,
```

Comment:
nitpick: indent fix

run `make pylint` to catch these issues

Reply:
Agreed - fixed lint indentation errors.